foundation/foundation-mda/src/main/resources/templates/data-lineage-consumer/data-lineage-consumer.pom.xml.vm

```diff
@@ -13,13 +13,6 @@
 
   <dependencyManagement>
     <dependencies>
-      <dependency>
-        <groupId>com.boozallen.aissemble</groupId>
-        <artifactId>bom-component</artifactId>
-        <version>${version.aissemble}</version>
-        <type>pom</type>
-        <scope>import</scope>
-      </dependency>
```

Comment:
Think we discussed this previously but wanted to verify, we plan on creating a baton migration for all these downstream template changes in a later ticket, correct?

Reply:
Correct. Follow-on work will include migrating files derived from templates and applying the known Java 11 -> Java 17/ Quarkus 2 -> Quarkus 3 fixes.